cmd/queue/main.go

```diff
@@ -333,12 +378,15 @@ func buildBreaker(logger *zap.SugaredLogger, env config) *queue.Breaker {
 	return queue.NewBreaker(params)
 }
 
-func supportsMetrics(ctx context.Context, logger *zap.SugaredLogger, env config) bool {
+func supportsMetrics(ctx context.Context, logger *zap.SugaredLogger, env config, enableTLS bool) bool {
+	// Metrics needs to be registered on either TLS server or non-TLS server. Give it away to the TLS server.
```

Comment:
This says "give it away to the TLS server", but it looks like the TLS server will report that it doesn't support metrics.

Maybe this should be "keep it on HTTP because..."